.github/actions/azviz/README.md

```diff
@@ -0,0 +1,90 @@
+# AzViz (Azure Visualizer) action
+Note: This GitHub Action is imported from: https://github.com/josiahsiegel/azviz-action
+
+## ☕ Please donate to [AzViz Developer](https://github.com/PrateekKumarSingh/AzViz#readme)
+
+![](https://github.com/PrateekKumarSingh/AzViz/blob/master/img/themeneon.jpg)
+
+## Synopsis
+
+[AzViz](https://github.com/PrateekKumarSingh/AzViz) for [GitHub actions](https://github.com/marketplace?type=actions)!
+
+## Inputs
+
+### Required
+
+```yml
+inputs:
+  resource-group:
+    description: Comma-seperated resource group list
+    required: true
+  out-file:
+    description: Graph export path
+    required: true
+    default: output/viz.svg
+  sub-name:
+    description: Azure subscription name
+    required: true
+    default: Pay-As-You-Go
+```
+
+### Optional
+
+```yml
+  theme:
+    description: Graph theme (dark, light, neon)
+    required: false
+    default: neon
+  depth:
+    description: Level of Azure Resource Sub-category to be included in vizualization (1 or 2)
+    required: false
+    default: '1'
+  verbosity:
+    description: Level of information to included in vizualization (1 or 2)
+    required: false
+    default: '1'
+  format:
+    description: Graph format (png or svg)
+    required: false
+    default: svg
+  direction:
+    description: Direction in which resource groups are plotted on the visualization (left-to-right or top-to-bottom)
+    required: false
+    default: top-to-bottom
+  exclude-types:
+    description: Exclude resources via string search
+    required: false
+    default: '*excludethisthing1,excludethisthing2*'
+  splines:
+    description: Controls how edges appear in visualization. ('spline', 'polyline', 'curved', 'ortho', 'line')
+    required: false
+    default: spline
+```
+
+## Quick start
+
+`sample_min_workflow.yml`
+```yml
+jobs:
+  generate-viz:
+      runs-on: ubuntu-latest
+      steps:
+        - name: Login to Azure
+          uses: azure/login@v1
+          with:
+            creds: ${{ secrets.SERVICE_PRINCIPAL_CREDS }}
+            enable-AzPSSession: true
+        - uses: CDCgov/prime-reportstream/.github/actions/azviz@663e24299a6336f1ff8dbddadfac1ba5d462f731aaa
```

Comment:
We should version pin, not SHA.

Reply:
I totally agree. Let me ask you something:
Which version should we use? This is not an actual GitHub repo but rather an imported repo inside another repo and not as a sub-repo. So, unless it's decided to define an abstracted version-id, there is no version.
This is a README.md file and this change is intended to keep track of what hardcoded hash commit is defined within the CI/CD pipeline in the project's GitHub Action.